admin/view-translations-term.php

```diff
@@ -31,19 +34,23 @@
 
 		// Look for any existing translation in this language
 		// Take care not to propose a self link
-		$translation = 0;
+		$translation = null;
 		if ( isset( $term_id ) && ( $translation_id = $this->model->term->get_translation( $term_id, $language ) ) && $translation_id != $term_id ) {
 			$translation = get_term( $translation_id, $taxonomy );
 		}
 		if ( ! empty( $from_term_id ) && ( $translation_id = $this->model->term->get( $from_term_id, $language ) ) && ! $this->model->term->get_translation( $translation_id, $lang ) ) {
 			$translation = get_term( $translation_id, $taxonomy );
 		}
 
+		$add_link = '';
+		$link     = '';
 		if ( isset( $term_id ) ) { // Do not display the add new link in add term form ( $term_id not set !!! )
-			$link = $add_link = $this->links->new_term_translation_link( $term_id, $taxonomy, $post_type, $language );
+			$link = $this->links->new_term_translation_link( $term_id, $taxonomy, $post_type, $language );
+			$add_link = $link;
 		}
```

Comment:
Let's simplify:
```php
		$add_link = isset( $term_id ) ? $this->links->new_term_translation_link( $term_id, $taxonomy, $post_type, $language ); // Do not display the add new link in add term form ($term_id not set).			
		$link = $add_link;

```

Reply:
Done in ace25b6b7e487239403ab9f0369325565fec4153